src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newSeparator = toFormat.slice(-1);
+  const formatDate = date.split(fromFormat[3]);
+  const newArr = [];
+
+  let day, month, year;
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    const element = formatDate[i];
+
+    if (fromFormat[i].includes('Y')) {
+      year = element;
+    }
+
+    if (fromFormat[i].includes('M')) {
+      month = element;
+    }
+
+    if (fromFormat[i].includes('D')) {
+      day = element;
+    }
```

Comment:
it's better to use switch case statement here instead of multiple if blocks

Reply:
> it's better to use switch case statement here instead of multiple if blocks

How can I do this without using 'includes'? Or how can I use this method in a switch case?